pilot/pkg/networking/core/v1alpha3/cluster_builder.go

```diff
@@ -477,20 +477,22 @@ func (cb *ClusterBuilder) buildInboundClusterForPortOrUDS(clusterPort int, bind
 				},
 			},
 		}
-		// add extra source addresses to cluster builder
-		var extraSrcAddrs []*core.ExtraSourceAddress
-		for _, extraBdIP := range cb.passThroughBindIPs[1:] {
-			extraSrcAddr := &core.ExtraSourceAddress{
-				Address: &core.SocketAddress{
-					Address: extraBdIP,
-					PortSpecifier: &core.SocketAddress_PortValue{
-						PortValue: uint32(0),
+		if features.EnableDualStack && len(cb.passThroughBindIPs) > 1 {
```

Comment:
Is the condition same as 
`if features.EnableDualStack && cb.supportsIPv4 && cb.supportsIPv6`

Reply:
Hi @hzxuzhonghu,  there may be a situation: the k8s cluster is deployed in a single stack, and enable DualStack when installing Istio, then, one of `cb.supportsIPv4` and `cb.supportsIPv6` may be false. So question is: should we consider this situation?